client/app/telecom/telephony/billingAccount/administration/addGroup/telecom-telephony-billing-account-administration-add-group.html

```diff
@@ -11,58 +11,46 @@ <h1 data-translate="telephony_add_group_title"></h1>
         <oui-spinner></oui-spinner>
     </div>
 
-    <div class="telecom-section-content"
+    <div class="telecom-section-content col-lg-11"
```

Comment:
You are not allowed to apply `col*` because your parent `<div>` element doesn't have `.row` as a class.

Please refer to: https://github.com/twbs/bootlint/wiki/E014